packages/manager/src/components/Dialog/Dialog.tsx

```diff
@@ -118,30 +116,27 @@ const Dialog: React.FC<DialogProps> = (props) => {
     >
       <Grid container alignItems="center">
         <div className={classes.sticky}>
-          <Grid>
-            <Typography
-              variant="h2"
-              id={titleID}
-              data-qa-drawer-title={title}
-              data-qa-dialog-title={title}
-            >
-              {title}
-            </Typography>
-          </Grid>
-          <Grid>
-            <Button
-              buttonType="secondary"
-              onClick={props.onClose as (e: any) => void}
-              className={classes.button}
-              data-qa-close-drawer
-              aria-label="Close"
-            >
-              <Close />
-            </Button>
-          </Grid>
+          <Typography
+            variant="h2"
+            id={titleID}
+            data-qa-drawer-title={title}
+            data-qa-dialog-title={title}
+          >
+            {title}
+          </Typography>
+
+          <Button
+            buttonType="secondary"
+            onClick={props.onClose as (e: any) => void}
+            className={classes.button}
+            data-qa-close-drawer
+            aria-label="Close"
+          >
+            <Close />
+          </Button>
         </div>
         {titleBottomBorder && <hr className={classes.titleBottomBorder} />}
-        <Grid container>
+        <Grid container sx={{ margin: '0 32px 0 32px' }}>
```

Comment:
```suggestion
        <Grid container sx={{ margin: '0 32px' }}>
```
We shouldn't need to repeat all four values since the top/bottom and left/right are the same on each side.